Add first two github issue templates

Atticus29 opened the pull request:
# Description

A reproduction of https://github.com/Symbiota/Symbiota/pull/3144 (cherry pick of eeedfd50d), but into the Hotfix-3.3.13 branch per request from @GregoryPost .

# Pull Request Checklist:

# Pre-Approval

- [x] There is a description section in the pull request that details what the proposed changes do. It can be very brief if need be, but it ought to exist
- [ ] Hotfixes should be branched off of the `master` branch and PR'd using the **squash & merge** option into the `Hotfix-x.x.x` branch.
- [ ] Features and backlog bugs should be merged into the `Development` branch, **NOT** `master`
- [ ] All new text is preferably internationalized (i.e., no end-user-visible text is hard-coded on the PHP pages), and the [spreadsheet tracking internationalizations](https://docs.google.com/spreadsheets/d/133fps9w2pUCEjUA6IGCcQotk7dn9KvepMXJ2IWUZsE8/edit?usp=sharing) has been updated either with a new row or with checkmarks to existing rows
- [x] There are no linter errors
- [ ] New features have responsive design (i.e., look aesthetically pleasing both full screen and with small or mobile screens)
- [x] [Symbiota coding standards](https://docs.google.com/document/d/1-FwCZP5Zu4f-bPwsKeVVsZErytALOJyA2szjbfSUjmc/edit?usp=sharing) have been followed
- [ ] If any files have been reformatted (e.g., by an autoformatter), the reformat is its own, separate commit in the PR
- [ ] Comment which GitHub issue(s), if any does this PR address
- [ ] If this PR makes any changes that would require additional configuration of any Symbiota portals outside of the files tracked in this repository, make sure that those changes are detailed in [this document](https://docs.google.com/document/d/1T7xbXEf2bjjm-PMrlXpUBa69aTMAIROPXVqJqa2ow_I/edit?usp=sharing)

# Post-Approval

- [ ] It is the code author's responsibility to merge their own pull request after it has been approved
- [ ] If this PR represents a merge into the `Development` branch, remember to use the **squash & merge** option
- [ ] If this PR represents a merge into the `hotfix` branch, remember to use the **squash & merge** option
- [ ] If this PR represents a merge from the `Development` branch into the master branch, remember to use the **merge** option
- [ ] If this PR represents a merge from the `hotfix` branch into the `master` branch use the **merge** option (not squashed)
  - [ ] a subsequent PR from `master` into `Development` should be made with the **merge** option (i.e., no squash)
  - [ ] **Immediately** delete the `hotfix` branch and create a new `hotfix` branch
  - [ ] increment the Symbiota version number in the symbase.php file and commit to the `hotfix` branch
- [ ] If the dev team has agreed that this PR represents the last PR going into the `Development` branch before a tagged release (i.e., before an imminent merge into the master branch), make sure to notify the team and [lock the `Development` branch](https://github.com/BioKIC/Symbiota/settings/branches) to prevent accidental merges while QA takes place. Follow the release protocol [here](https://github.com/BioKIC/Symbiota/blob/master/docs/release-protocol.md)
- [ ] Don't forget to delete your feature branch upon merge. Ignore this step as required

Thanks for contributing and keeping it clean!

Atticus29 committed:
add first two github issue templates. Cherry pick of eeedfd50d

MuchQuak commented on .github/ISSUE_TEMPLATE/newfeature.md:
```diff
@@ -0,0 +1,53 @@
+
+## Github Issue/Projects Fields
+
+- [ ] Populate label (e.g., bug, enhancement, etc.)
```
A bit pedantic but to avoid confusion 'bug' should be remove/replace with something here.

Atticus29 replied:
I'm confused what you mean here. "Bug" is one of the labels:
<img width="1582" height="946" alt="Screenshot 2025-12-22 at 10 34 55 AM" src="https://github.com/user-attachments/assets/f5ba1255-702c-4bce-a4f6-202d2cd33adc" />

Are you recommending that I replace with something like, "up for grabs", just so that that there's no ambiguity surrounding what I mean by, "label"?

MuchQuak replied:
I guess I was just pointing out that the new feature template would never have the bug tag so it would be confusing to mention it on the checklist. Unless this template is also used for bugs that you can disregard this whole point.

MuchQuak reviewed (approved):
I don't see anything wrong with this and I would just defer to @themerekat and @ljwalker since they are more in tune how users report problems and how help desk issues come in.

ljwalker committed:
Add link to ticket

- Added bullet to remind issue creator to include link (or ticket #) to original osTicket record
- Made minor punctuation and spacing edits

ljwalker committed:
Add link to ticket

Same as above

ljwalker committed:
Update helpdesk.md

Atticus29 committed:
change helpdesk to generic bug github issue template

Atticus29 committed:
remove crufty second Optional section header
